client/state/data-layer/wpcom/read/lists/index.js

```diff
@@ -57,11 +58,10 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/index.js', {
 						} ),
 					];
 				}
-				// NOTE: Add better handling for unexpected response format here.
-				errorNotice( translate( 'List could not be created, please try again later.' ) );
+				errorNotice( translate( 'Unable to create new list.' ) );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Unable to create a new list.' )` *ES Score: 8*